packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -13,7 +13,7 @@ include = [
     "../collateral_pools/collateral_pool_1/testnet.toml",
     "../orders_gateway/testnet.toml",
 ]
-version = "1.0.2"
+version = "1.0.3"
```

Comment:
can you please update this PR as well? https://github.com/Reya-Labs/reya-deployments/pull/40

Reply:
done